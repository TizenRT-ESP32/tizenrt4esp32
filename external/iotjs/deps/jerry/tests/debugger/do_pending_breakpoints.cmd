break f
y
list
n
n
c
c
