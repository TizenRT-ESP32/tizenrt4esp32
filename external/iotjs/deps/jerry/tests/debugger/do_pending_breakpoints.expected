Connecting to: localhost:5001
Stopped at tests/debugger/do_pending_breakpoints.js:15
(jerry-debugger) break f
No breakpoint found, do you want to add a pending breakpoint? (y or [n])
Pending breakpoint at f()
(jerry-debugger) list
=== Pending breakpoints ===
 1: f() (pending)
(jerry-debugger) n
out: pending-breakpoints
Stopped at tests/debugger/do_pending_breakpoints.js:17
(jerry-debugger) n
Breakpoint 2 at <unknown>:1 (in f() at line:1, col:1)
Stopped at tests/debugger/do_pending_breakpoints.js:19
(jerry-debugger) c
Stopped at breakpoint:2 <unknown>:1 (in f() at line:1, col:1)
(jerry-debugger) c
